Makefile

```
@@ -177,6 +177,9 @@ crds: $(CONTROLLER_GEN) $(YQ)
 	@echo Updating CRD manifests
 	@build/crds/build-crds.sh $(CONTROLLER_GEN) $(YQ)
 
+helm-gen-rbac: ## generate RBAC from Helm charts
```

Do we need the helm prefix?
```suggestion
gen-rbac: ## generate RBAC from Helm charts
```